### ISSUE TITLE
Revoke changes done to Category by users without permission

### ISSUE THREAD

**Issue by chandler05** [mojira/arisa-kt#411]
## The Problem
Users have found multiple ways to change this field
## Possible Solution
Revoke these changes unless the user is a mod+

**Comment by urielsalis** [comment]
Is this still a issue or its not worth checking with arisa?

**Comment by chandler05** [comment]
Yeah, we constantly are having to reset the category because a user figures out how to change it

**Comment by Jack-McKalling** [comment]
Also note I've seen it happen once during original submit of a report. So not just _changes_ to the field, also the originally submitted value should be monitored this way. I'm not sure how trivial or complicating that would make this.
